process_vanguard_bibs.py

```diff
@@ -97,12 +98,54 @@ def move_939_fatadb(record):
         record.remove_field(old_fld)
     for fld in record.get_fields("939"):
         fld.tag = "969"
+        record.remove_field(fld)
         record.add_ordered_field(fld)
 
 def do_SILSLA_15_bib(record, dbcode):
     delete_various_9xx(record)
     copy_001(record, dbcode)
     move_9xx(record)
+
+#SILSLA-16
+
+def delete_035_subfield(record):
+    """Delete 035 $9 where appropriate"""
+    for fld in record.get_fields("035"):
+        if fld["9"] != None and fld["9"] == 'ExL' and fld["a"] != None:
+            fld.delete_subfield("9")
+
+def delete_035(record):
+    """Delete 035 field where appropriate"""
+    for fld in record.get_fields("035"):
+        if fld["9"] != None and fld["9"] == 'ExL' and fld["a"] == None:
+            record.remove_field(fld)
+
+def move_035(record):
+    """Move value of 035 $9 to 992 $c"""
+    for fld in record.get_fields("035"):
+        if fld["9"] != None and fld["9"] != 'ExL':
+            sfld = copy.copy(fld["9"])
+            fld.delete_subfield("9")
+            fld_992 = Field(tag="992", indicators=[' ',' '], subfields=['c', sfld])
+            record.add_ordered_field(fld_992)
+        #only deleting $9 leaves some empty 035 fields
```

Comment:
Specs say that after moving 035 $9 to 992 $c, the source 035 should be deleted regardless of remaining content.
> MOVE each value in a $9 to a new 992 field in subfield $c (and delete the originating 035 field)